video/out/wayland_common.c

```diff
@@ -2098,6 +2132,37 @@ static int handle_round(int scale, int n)
     return (scale * n + WAYLAND_SCALE_FACTOR / 2) / WAYLAND_SCALE_FACTOR;
 }
 
+#if HAVE_FROG_PROTOCOLS
+static enum pl_color_primaries pl_primaries_to_frog_primaries(enum pl_color_primaries primaries)
```

Comment:
nit pick, could drop 2nd "transfer". We already use `pl_transfer_from_av` and `pl_transfer_to_av`, so, could be `pl_transfer_to_frog`

Reply:
I dropped the 2nd "primaries" too. It's clear enough.